### ISSUE TITLE
Samsung LCD panels

### ISSUE THREAD

**Issue by sagonzal** [andysworkshop/xmemtft#8]
First, I wanted to congratulate the author, Andy this is a tremendous work! thank you. Now, I'm having QVGA LCD panels from Samsung phones lying around and I would like to test xmemfft with them. I read the article in your blog https://andybrown.me.uk/2013/06/08/a-generic-16-bit-lcd-adaptor-for-the-arduino and I see you haven't tested them.
What do you think ? The board-to-board connector of this LCD panels seems very similar! 
Does it worth the effort to reverse engineer them ?

Thank you in advance for any help....

**Comment by andysworkshop** [comment]
Hi, firstly you'd need the schematics to find the connector pinout. I wouldn't be very hopeful though because all manufacturers migrated from the easy to control DBI interface to the much harder DSI interface years ago.

**Comment by sagonzal** [comment]
You are right, Display Serial Interface (DSI) is now everywhere (even  in the raspberry pi). So, to clearify for  myself, not every manufacturer has a standard DBI pin-out ?

again, thank you in advance...

**Comment by andysworkshop** [comment]
There is no standard DBI pinout on the connector. All the logical DBI functions will be present but I've found that their position on the connector will vary between models.

There's also the backlight and power pins that need to be found - ground is often the easiest because it's usually mapped to a fat trace on the flat connector. Then you can use your meter to buzz the other pins to find out any other grounds.
